### ISSUE TITLE
version 0.12.1 quarantined by Trend Micro Antivirus

### ISSUE THREAD

**Issue by HeatherUrry** [jasp-stats/jasp-issues#703]
* JASP version: 0.12.1
* OS name and version: Windows 10 Enterprise
* Analysis: Bayesian Rep Measures ANOVA (I think)
* Bug description: Trend Micro Antivirus placed JASP.exe in quarantine

![image](https://user-images.githubusercontent.com/30731182/80140121-82c77900-8575-11ea-9425-18ffef864615.png)

(The "Details" link in that screenshot links to [here](https://www.trendmicro.com/vinfo/us/threat-encyclopedia/search/unauthorized%20file%20encryption).)

**Comment by boutinb** [comment]
Hallo @HeatherUrry, does it happen only with the Bayesian Repeated Measures ANOVA?

**Comment by HeatherUrry** [comment]
Hi @boutinb, I don't know. I had only been using the Bayesian Rep Meas ANOVA module when JASP got quarantined. Since it got quarantined on the affected computer, I haven't been able to use it. (Apparently I don't have the administrative privilege to take it out of quarantine myself. Annoying.) I've now got JASP 0.12.1 running on a different computer but before I started using it, I made sure to put JASP.exe on the antivirus software exception list so it wouldn't get quarantined too. So, that installation is not a good place to put your question to the test.

**Comment by boutinb** [comment]
That is really weird, because we have a certificate that Windows trusts.
We have just released a new version 0.12.2. Can you have a look if it still occurs with this version?

**Comment by HeatherUrry** [comment]
Hi Bruno,

I downloaded and installed 0.12.2 on the laptop on which JASP.exe got quarantined yesterday. The installation worked just fine and I was able to open up an existing .jasp file. However, when I was trying to sync the csv data file after having made an edit to it, JASP.exe for 0.12.2 got quarantined by Trend Micro Antivirus. It displayed the same message about the threat being unauthorized file encryption. (This was both a different JASP module and a different data file compared to when the quarantine happened yesterday with 0.12.1.) 

No idea if this is an issue on the JASP side or something weird about my laptop or Trend Micro Antivirus. I'm running Windows 10 Enterprise, which was installed via a bootcamp partition on a MacBook Air. It works fine on the other computer after having told Trend Micro Antivirus to allow JASP.exe.

Best,

Heather
